tailwind.config.js

```diff
@@ -1,41 +1,34 @@
+const plugin = require('tailwindcss/plugin');
+
 module.exports = {
   theme: {
+    /**
+     * Themeable
+     */
     borderRadius: {
       none: '0',
       1: "var(--calcite-border-radius)",
       half: '50%',
       full: '100%',
     },
     boxShadow: {
+      // assets/styles/_shadow
       '1-sm': 'var(--calcite-shadow-1-press)',
       '1': 'var(--calcite-shadow-1)',
       '1-lg': 'var(--calcite-shadow-1-hover)',
       '2-sm': 'var(--calcite-shadow-2-press)',
       '2': 'var(--calcite-shadow-2)',
       '2-lg': 'var(--calcite-shadow-2-hover)',
+      'border-bottom': 'var(--calcite-shadow-border-bottom)',
+      'border-active': 'var(--calcite-shadow-border-active)', 
```

Comment:
The default config provides a similar utility, `shadow-outline`, would it be less confusing to rename this? Also, should we avoid introducing pseudo-classes into utility names? The variants generated for these utilities might add some confusion i.e. `active:shadow-border-active` and `hover:shadow-border-active` vs. `active:shadow-outline`.

Reply:
Thanks @jrowlingson.
Truth is, I'm using "active" here incorrectly. This style would be used for a "current" state, e.g. when a value-list-item hs been "selected". 

We could use "selected". I don't know if that being the property used on a select option would be more or less confusing. Thoughts?